test/MUXControlsTestApp/MSTest/MUXControlsTestApp.csproj

```diff
@@ -36,6 +36,11 @@
       <SubType>Designer</SubType>
     </AppxManifest>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Win2D.uwp">
+      <Version>1.23.0</Version>
+    </PackageReference>
+  </ItemGroup>
```

Comment:
I suspect this will be a problem in the OS build, what took a dependency on Win2D?

Reply:
AnimatedVisualPlayer_TestUI LottieLogo.cs which is the content source for AnimatedVisualPlayer. This LottieLogo.cs has CanvasGeometry type which requires Win2D package. I recalled I have fixed this version to 1.21.0 or similar, not sure it is still 1.23.0. Let me double check and make sure it compiled in local.

Reply:
Is Win2d available in razzle? This code will be compiling in the OS repo.

Reply:
Thanks to Simeon for his updating content source LottieLogo.cs which do not need to refer to Win2D.uwp package any more. This LottieLogo.cs has been updated and do not need to reference to Win2D any longer. And Win2D package is uninstalled.